config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
Curious, why did you choose `hybrid`? 🤔  As far as I know, it's an intermediate step when transitioning between the other two.

Reply:
I think so this change was a part of trying to debug the cookies issue. I was trying a few things and this was one of them. Reverted it. Thanks for pointing